### PR TITLE
dto added succesfully

### PR DESCRIPTION
**Pull request by sagarb2722**

### CODE REVIEW

File: src/main/java/com/ty/password/dto/User.java [hunk 1]
```diff
@@ -1,5 +1,49 @@
 package com.ty.password.dto;
 
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@EnableWebMvc
```
Comment (GSTharun) [inline src/main/java/com/ty/password/dto/User.java]: Its Entity class please change to @Entity

File: src/main/java/com/ty/password/dto/User.java [hunk 2]
```diff
@@ -1,5 +1,49 @@
...
 public class User {
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
+	private int id;
+	private String name;
+	private String address;
+	private String email;
+	private String password;
```
Comment (sahanaagowda) [inline src/main/java/com/ty/password/dto/User.java:19]: add phone number as attribute

File: src/main/java/com/ty/password/dto/User.java (hunk 1 above)
Comment (sagarb2722) [reply src/main/java/com/ty/password/dto/User.java]: yup

File: src/main/java/com/ty/password/dto/User.java [hunk 3]
```diff
@@ -1,5 +1,49 @@
...
+		return email;
+	}
+	public void setEmail(String email) {
+		this.email = email;
+	}
+	public String getPassword() {
+		return password;
+	}
+	public void setPassword(String password) {
+		this.password = password;
```
Comment (vvviin) [inline src/main/java/com/ty/password/dto/User.java:55]: override tostring method

**Review by GSTharun — APPROVED** [review]

**Review by vvviin — APPROVED** [review]

**Review by sahanaagowda — APPROVED** [review]

**Review by GSTharun — APPROVED** [review]

**Review by sahanaagowda — APPROVED** [review]

**Review by vvviin — APPROVED** [review]

**Pull request merged**